icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -4014,6 +4014,7 @@ void RBBITest::RunMonkey(BreakIterator *bi, RBBIMonkeyKind &mk, const char *name
                 errln("breakPos > testText.length()");
             }
             expectedBreaks[breakPos] = 1;
+            expectedCount++;
```

Comment:
Wow...!
Would you mind taking a look at whether the Java test code has this same bug?